scripts/ci-cache-files.sh

```diff
@@ -9,6 +9,7 @@ EC_VERSION=${EC_VERSION:-}
 K0S_VERSION=${K0S_VERSION:-}
 AWS_REGION="${AWS_REGION:-us-east-1}"
 S3_BUCKET="${S3_BUCKET:-dev-embedded-cluster-bin}"
+CACHE_BINS=${CACHE_BINS:-1}
```

Comment:
could you change this to UPLOAD_BINARIES perhaps in a follow up pr? really the file should be renamed as its not caching them. It is uploading them so that the release can be installed remotely.

Reply:
yes, i can update that in a separate pr